src/sb/app/project/ui.cljc

```diff
@@ -126,6 +127,23 @@
      [entity.ui/persisted-attr project :entity/admission-policy props])]
   )
 
+;; TODO: taken from board.ui, merge into common component if sensible
```

Comment:
Agree, let's move this to a shared namespace as it's a very general implementation